modules/caddypki/acmeserver/acmeserver.go

```diff
@@ -122,8 +127,8 @@ func (ash *Handler) Provision(ctx caddy.Context) error {
 			},
 		},
 		DB: &db.Config{
-			Type:       "badger",
-			DataSource: dbFolder,
+			Type:       "bagerV2",
```

Comment:
Is this a typo? Should be `badgerV2`, no?

Reply:
Yes. Yes it is a typo. Fixing now.